src/dealDashboard/dealMenubar/dealMenubar.ts

```diff
@@ -19,12 +19,12 @@ export class DealMenubar {
     this.eventAggregator.publish("handleInfo", "Deal URL copied");
   }
 
-  @computedFrom("deal.status", "deal.isCancelled")
+  @computedFrom("deal.isUserProposalLead", "deal.isCancelled", "deal.isExecuted")
   get canEdit() {
     return this.deal.isUserProposalLead && !this.deal.isCancelled && !this.deal.isExecuted;
   }
 
-  @computedFrom("deal.status", "deal.isCancelled")
+  @computedFrom("deal.isUserProposalLead", "deal.isCancelled", "deal.isExecuted")
   get canCancel() {
     return this.deal.isUserProposalLead && !this.deal.isCancelled;
```

Comment:
why need `.isExecuted` when the getter's return does not use `isExecuted`?

Reply:
should be: "deal.isUserProposalLead", "deal.isCancelled" (removed "deal.isExecuted")